test/mongodb_test.js

```diff
@@ -230,7 +230,7 @@ vows.describe('mongodb').addBatch({
           return this.callback(err);
         }
 
-        db.addIndex('event', 'event_title', 'title', false, function(err, data) {
```

Comment:
Last thing missing here: You have deleted the old tests, but to actually cover the old and new functionality, the old and the new tests have to coexist, rather than one gets rewritten.